tgui/packages/tgui/interfaces/PlaneMasterDebug/types.ts

```diff
@@ -0,0 +1,129 @@
+import { Position } from './../common/Connections';
+
+export type PlaneDebugData = {
+  mob_name: string;
+  mob_ref: string;
+  our_ref: string;
+  tracking_active: boolean;
+  enable_group_view: boolean;
+  our_group: string;
+  present_groups: string[];
+  planes: PlaneData[];
+};
+
+export type PlaneData = {
+  name: string;
+  documentation: string;
+  plane: number;
+  offset: number;
+  real_plane: number;
+  renders_onto: number[];
+  blend_mode: string;
+  color: string | number[];
+  alpha: number;
+  render_target: string;
+  force_hidden: boolean;
+  relays: RelayData[];
+  filters: FilterData[];
+};
+
+export type RelayData = {
```

Comment:
Do these need to be exported? What's the minimum that could be exported for this file?

Reply:
Every single one of the types.ts is used either implicitly (Data types) or explicitly (everything else)

Reply:
in that case, data types can just be `type RelayData`, you only need to export the parent